Kickstarter-iOS/DataSources/SearchDataSource.swift

```diff
@@ -8,13 +8,19 @@ internal final class SearchDataSource: ValueCellDataSource {
     case popularTitle
     case projects
     case noResults
+    case loading
   }
 
   internal func popularTitle(isVisible visible: Bool) {
     self.set(cellIdentifiers: visible ? ["MostPopularCell"] : [],
              inSection: Section.popularTitle.rawValue)
   }
 
+  internal func loader(isVisible visible: Bool) {
+    self.set(cellIdentifiers: visible ? ["SearchLoadingStateCell"] : [],
+             inSection: Section.loading.rawValue)
+  }
```

Comment:
it doesnt seem like we are using the search loading cell and instead using the background view and/or the table header view. is that correct?